npc/other/Global_Functions.txt

```diff
@@ -159,11 +159,8 @@ function	script	F_Rand	{
 //== Function F_Sex ========================================
 // Returns 1st argument if female, 2nd argument otherwise
 // Example: mes callfunc("F_Sex","What a beautiful lady!","What a handsome man!");
-function	script	F_SexMes	{
-	if (Sex == SEX_MALE)
-		return getarg(0);
-	else
-		return getarg(1);
+function	script	F_Sex	{
+	return getarg(Sex == SEX_MALE);
```

Comment:
Since there is a constant for Sex, I don't see the need for this function at all. Current functions using this function could be replaced with the constant in a new PR.